src/sdp/data.rs

```diff
@@ -670,6 +670,28 @@ impl MediaLine {
                 _ => {}
             }
         }
+
+        let simulcast_groups = self.attrs.iter().find_map(|a| match a {
+            MediaAttribute::Simulcast(s) => {
+                if is_send {
+                    Some(&s.send)
+                } else {
+                    Some(&s.recv)
+                }
+            }
+            _ => None,
+        });
+
+        // The order of the `a=ssrc-group:FID` lines map to the declared order of the `a=simulcast`
+        // line. For example
+        // a=simulcast:recv l;m;h
+        // a=ssrc-group:FID 1 2
+        // a=ssrc-group:FID 3 4
+        // a=ssrc-group:FID 5 6
+        //
+        // maps 1 as the SSRC for rid l, 2 as the RTX SSRC for rid l, 3 as the SSRC for rid m, 4 as
+        // the RTX SSRC for rid m and so on.
+        let mut group_count = 0;
```

Comment:
This is doubling up on functionality that already exists. See `simulcast()` call.

Reply:
I don't think so, this block

```rust

        let simulcast_groups = self.attrs.iter().find_map(|a| match a {
            MediaAttribute::Simulcast(s) => {
                if is_send {
                    Some(&s.send)
                } else {
                    Some(&s.recv)
                }
            }
            _ => None,
        });
```

from above could be replaced with a call to `simulcast`, but mapping the `ssrc-group` lines to the relevant `rid` isn't performed by it